docs/app/pages/css/css-sections/icons/ux-icons/icon-preview/icon-preview.component.html

```diff
@@ -0,0 +1,64 @@
+<button uxFocusIndicator
```

Comment:
I'm noticing some strange hover behaviour. The hover background colour now stretches the full width which is as expected, however hovering the mouse over the area does not show the hover effect until the mouse is near the text:

[![Image from Gyazo](https://i.gyazo.com/862c829911bc5bcc558a96bfe6c95dd8.gif)](https://gyazo.com/862c829911bc5bcc558a96bfe6c95dd8)

Reply:
Mouse over area is now full width of button